app/assets/stylesheets/components/dropdowns.scss

```diff
@@ -2,9 +2,11 @@
 
 .crayons-dropdown {
   position: absolute;
+  top: 100%;
```

Comment:
Turns out most of our dropdown are positioned this way so it made sense to have it there.